.github/workflows/labeler.yml

```
@@ -0,0 +1,12 @@
+name: "Pull Request Labeler"
+on:
+- pull_request_target
+
+jobs:
+  labeler:
+    permissions:
+      contents: read
+      pull-requests: write
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/labeler@v5
```

YAMLlint (1.37.1)

[error] wrong indentation: expected 6 but found 4

(indentation)